mne/io/base.py

```diff
@@ -1997,6 +2004,16 @@ def _write_raw(fname, raw, info, picks, fmt, data_type, reset_range, start,
 
         pos_prev = pos
 
+    if raw.annotations is not None:
+        start_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MIN, raw.annotations.onset)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MAX, raw.annotations.duration)
```

Comment:
Right, `duration` does not. But look at what you're writing -- a `tmin` and `tmax`. Thus it seems like you should actually write a `tmin` and `tmax`, not `onset` and `duration`. In other words, I'm fine with storing it as `onset` / `duration` in the `Annotation` class, but doing it that way in the file by using `tmin` and `tmax` tags doesn't make much sense to me.